app/helpers/activity_helper.rb

```diff
@@ -30,9 +30,4 @@ def user_can_add_transaction?(user, activity)
     return true if user.delivery_partner? && activity.project?
     user.administrator? || user.fund_manager?
   end
-
-  def user_can_add_budget?(user, activity)
```

Comment:
Could we squash this into 7f21cdfd9ad4a59eabaf7a7e2c762aed0ee56b07 which added this method in the commit before to clear up the git history?

Reply:
This commit has been dropped completely